webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
```

Comment:
Whatever you pull in you will risk (now, in the future or in an extension) getting stats objects with references to stats objects that are not in the returned set. I think we just have to accept that since recursively exploring the stats graph is IMO a mistake.

But you're right, if we make the rule about pulling in "any stats object referencing or referenced by the track stats" that might oblige us to pull in stuff we don't like if we update the spec with new references.